modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TaskHeaderActions.tsx

```diff
@@ -13,37 +13,63 @@
  */
 
 import ClayButton from '@clayui/button';
-import {useNavigate} from 'react-router-dom';
+import {useEffect, useState} from 'react';
+import {useNavigate, useOutletContext} from 'react-router-dom';
 import {KeyedMutator} from 'swr';
 
-import AssignModal from '../../components/AssignModal';
 import useFormModal from '../../hooks/useFormModal';
 import i18n from '../../i18n';
-import {TestrayTask, UserAccount, testrayTaskImpl} from '../../services/rest';
+import {
+	TestrayTask,
+	TestrayTaskUser,
+	testrayTaskImpl,
+} from '../../services/rest';
+import TestflowAssignUserModal from './modal';
 
-type TaskHeaderActionsProps = {
-	TestrayTask: TestrayTask;
-	mutateTask: KeyedMutator<TestrayTask>;
+type OutletContext = {
+	mutateTask: KeyedMutator<any>;
```

Comment:
Missing the type

```suggestion
	mutateTask: KeyedMutator<TestrayTask>;
```

Reply:
We fixed it in one place and it broke in another, I suggest we change the any when we refactor the outlet @kevenleone